src/us/deathmarine/luyten/MainWindow.java

```diff
@@ -38,7 +38,7 @@
 public class MainWindow extends JFrame {
 	private static final long serialVersionUID = 5265556630724988013L;
 
-	private static final String TITLE = "Luyten";
+	private static final String TITLE = "Luyten 2";
```

Comment:
This wasn't intended to be in the pull request.. It was just for testing purposes.